tests/authentication_tests/test_interactive_authentication.py

```diff
@@ -319,7 +319,24 @@ def test_sso_enabled_site(self, *mocks):
         on an SSO-enabled site.
         """
         handler = console_authentication.ConsoleLoginHandler(fixed_host=False)
-        with self.assertRaises(ConsoleLoginWithSSONotSupportedError):
+        with self.assertRaises(ConsoleLoginNotSupportedError):
```

Comment:
I think you should still catch the previous exception type to make sure its still available. Please also add a comment explaining the backward compatiblity check so that a future reader is not tempted to change the test to the new class.

Reply:
Created a new test for that sole purpose.